content/blog/dplyr-1-0-0/index.Rmd

```diff
@@ -0,0 +1,116 @@
+    In `summarise()`, a single summary expression can now return create both
+    multiple rows and multiple columns. This significantly increases its power
+    and flexibility.
+
+-   [`select()`, `rename()`, and (new) `relocate()`](/blog/2020/03/dplyr-1-0-0-select-rename-relocate/). 
+    `select()` and `rename()` can now select by position, name, function of
+    name, type, and any combination of above. A new `relocate()` function 
+    makes it easy to change the position of columns.
+
+-   [Working `across()` columns](/blog/2020/04/dplyr-1-0-0-colwise/).
+    A new `across()` function makes it much easier to apply the same operation
+    to multiple columns. It supersedes the `_if()`, `_at()`, and `_all()`
+    function variants.
+
+-   [Working within rows](/blog/2020/04/dplyr-1-0-0-rowwise/).
+    `rowwise()` has been renewed and revamped to make it easier to perform
+    operations row-by-row. This makes it much easier to solve problems that
+    previously required `lapply()`, `map()`, or friends.
+
+-   [The role of the vctrs package](/blog/2020/04/dplyr-1-0-0-and-vctrs/).
+    dplyr now makes heavy use of [vctrs](http://vctrs.r-lib.org/) behind
+    the scenes. This brings with it greater consistency and (hopefully!)
+    more useful error messages.
+
+-   [Last minute additions](/blog/2020/05/dplyr-1-0-0-last-minute-additions/)
+    `summarise()` now allows you to control how its results are grouped,
+    and there's a new family of functions designed for modifying rows.
+
+You can see the full list of changes in the [release notes](https://github.com/tidyverse/dplyr/releases/tag/v1.0.0).
+
+## New logo
+
+dplyr has a new logo thanks to the talented [Allison Horst](https://allisonhorst.github.io)!
+
+![New dplyr logo](dplyr.png){width=250px}\ 
+
+(Stay tuned for details about how to get this sticker on to your laptop. We have some exciting news coming up!)
+
+## A small teaser
+
+The best way to find out all the cool new features dplyr has to offer is to read through the blog posts linked to above. But thanks to inspiration from [Daniel Anderson](https://twitter.com/datalorax_/status/1258208502960422914) here's one example of fitting two different models to here are bunch of cool features in one single example:
+
+```{r}
+library(dplyr, warn.conflicts = FALSE)
+
+models <- tibble::tribble(
+  ~model_name,    ~ formula,
+  "length-width", Sepal.Length ~ Petal.Width + Petal.Length,
+  "interaction",  Sepal.Length ~ Petal.Width * Petal.Length
+)
+
+iris %>% 
+  nest_by(Species) %>% 
+  left_join(models, by = character()) %>% 
+  rowwise(Species, model_name) %>% 
+  mutate(model = list(lm(formula, data = data))) %>% 
+  summarise(broom::glance(model))
+```
+
+Note the use of:
+
+* The new `nest_by()`, which generate a nested data frame where each row
+  represents one subgroup.
+
+* In `left_join()`, `by = character()` now performs a Cartesian product, 
+  generates every combination of subgroup and model.
```

Comment:
_"…performs a Cartesian product, and generates…"_

_"…performs a Cartesian product, generating…"_

Do either of these work for you? Or we could be wordy about it and do _…thereby generating…_, or something to that effect.